src/getStyles.js

```diff
@@ -1 +1,5 @@
-export const getStyles = (globalStyles, blockConfig) => ({});
```

Comment:
We don't have globalStyles anymore?

Reply:
No, the new block contract no longer supports it. It also seemed that every time they were referenced in the old contract it was completely superfluous, so there's no loss removing them.

Reply:
So are we removing the global styles panel from the SD?

Reply:
No. Cameron is working on how the global styles will interact with the button component, and the other styles there are really only needed for the basic global css that is added by the Site Designer and Renderer. Every time we saw them being used in the blocks, it was unnecessary redundancy.

Reply:
> So are we removing the global styles panel from the SD?

@santios-volusion No, the global styles will be used by the new components that we will be making, so we can leave the panel in. Blocks that need to hook into global styles can still do it through Components, they just can't access the styles object directly.